### PR TITLE
Routing and content layout

### PR DESCRIPTION
**Pull request by aniablaziak**

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-import CurrentMenu from "./current-menu/index";
-import DesignInfo from "./design-info/index";
-import EastInfo from "./east-info/index";
-import InterventionDesign from "./intervention-design/index";
-import LessonConfirmation from "./lesson-confirmation/index";
-import LessonIntro from "./lesson-intro/index";
-import PreviewPage from "./preview-page/index";
-import VegDish from "./veg-dish/index";
-import WhichLesson from "./which-lesson/index";
+import Home from "./screens/home";
```
Comment (errkk) [inline src/components/index.js]: i _think_ you don't have to type index if ya wanna save some letters :D

File: src/components/screens/before-intervention/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            brainstorming session with your team to come up with ideas and
+            finalise the name variations.
+          </StyledP>
+          <StyledP>
+            We have prepared a document to help get you started in your
+            brainstorming session. This can be downloaded here.
+          </StyledP>
+        </TextContainer>
+        <LeftButtonContainer>
+          <Link
```
Comment (errkk) [inline src/components/screens/before-intervention/index.js]: i think you can do styled components on the router things as well... something like: 
```
const StyledLink = styled(Link)`
	text-decoration: none;
	...
`;
```

File: src/components/index.js (hunk 1 above)
Comment (aniablaziak) [reply src/components/index.js]: I think you're absolutely right 😄

File: src/components/screens/before-intervention/index.js (hunk 2 above)
Comment (aniablaziak) [reply src/components/screens/before-intervention/index.js]: ohhh, cool!

**Pull request merged**